public/data/ShreyaPrasad1209.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Shreya Prasad",
+  "type": "personal",
+  "bio": "Software Engineer at Intuit | CS @IGDTUW",
+  "avatar": "https://github.com/ShreyaPrasad1209.png",
+  "links": [
+    {
+      "name": "Follow my Open Source on GitHub",
+      "url": "https://github.com/ShreyaPrasad1209",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/shreyacasmalert",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/-shreya-prasad/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on Medium",
+      "url": "https://medium.com/@shreyapd06",
+      "icon": "medium"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/shreyaprasad_/",
```

Comment:
The code looks good ✅.
I have a one doubt: 👇
You should give a look at your Instagram username.
can you confirm which one is correct, the previous one or the below one.
 ```suggestion
       "url": "https://www.instagram.com/shreyacasmalert_/",
```

Reply:
Both urls are incorrect, I've updated it in my latest commit.

Reply:
> Both urls are incorrect, I've updated it in my latest commit.

reviewed✅.
By the way, the username I have mentioned above is the same one that you have committed.